traffic_portal/app/src/common/modules/table/servers/TableServersController.js

```diff
@@ -548,6 +553,20 @@ var TableServersController = function(servers, $scope, $state, $uibModal, $windo
 		}
 	};
 
+	$scope.onQuickSearchChanged = function() {
+		$scope.gridOptions.api.setQuickFilter($scope.quickSearch);
+		localStorage.setItem("servers_quick_search", $scope.quickSearch);
+	};
+
+	$scope.onPageSizeChanged = function() {
+		const value = Number($scope.pageSize);
```

Comment:
I could be wrong, but since you guarded in the `localStorage` load and used `input[type=number]` I think this may not be necessary.

Reply:
maybe not but i recall it not working before the casting. casting a number to a number won't hurt so...

Reply:
i think it's because `input[type=number]` is still a number string

Reply:
Oh, in general if you access `value` you get back a `USVString` type, but I thought Angular bound form controls to `valueAsNumber` to avoid that for numeric inputs. I could be wrong, might be an Angular thing and not an AngularJS thing.